pkg/shell/superuser.jsx

```diff
@@ -46,6 +50,12 @@ class UnlockDialog extends React.Component {
         let body = null;
         let footer = null;
 
+        const auto_checkbox = (
+            <Checkbox onChange={this.props.change_remember_access_level_on_unlock}
+                      isChecked={this.props.remember_access_level_on_unlock}
+                      label={_("Use administrative mode when logging in")}
+                      description={_("Automatically switching to administrative mode may not work as expected, especially on remote hosts and when using 2FA")} />);
```

Comment:
s/and/or/ ? "Remote hosts" is a bit confusing, as the machine you are connecting to (to port 9090) is already likely "remote", but that's not what we mean here. Perhaps "especially with SSH keys or when using two-factor authentication"?

Reply:
What about:

"When logging in with a password, that password is automatically used to gain administrative access. This will only work if the login password is the same as the password that unlocks administrative access. This is often the case, but not when using 2FA, for example."

Reply:
Honestly I'd rather make it shorter than longer.. Somebody think of the chil^Wtranslators! :grinning: Maybe:

> Using your login password to automatically switch to administrative mode using may not work as expected, especially with SSH keys or 2FA.

Reply:
> Using your login password to automatically switch to administrative mode using may not work as expected, especially with SSH keys or 2FA.

Ok, I put that in.